.gitignore

```diff
@@ -27,3 +27,4 @@ dtm
 
 # .devstream
 .devstream/
+devstream.state
```

Comment:
`devstream.state` will located inside the `.devstream/` directory, so I think it's no need to put the `devstream.state` here.

Reply:
`devstream.state` now is moved out of the `.devstream/` folder.